common/pom.xml

```diff
@@ -213,6 +213,7 @@
                         <!-- Ignore exception classes -->
                         <exclude>**/AWSIncompatibleDataException*</exclude>
                         <exclude>**/AWSSchemaRegistryException*</exclude>
+                        <exclude>**/AWSSchemaNamingStrategy*</exclude>
```

Comment:
Why do we have to exclude this?

Reply:
Because it is in an interface and by default we don't call default methods.

Reply:
Can we test it by creating an implementation that doesn't call it?